sdks/python/src/opik/message_processing/emulation/local_emulator_message_processor.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+import logging
+from typing import List, Dict, Optional, Any, Union
+
+from opik.types import ErrorInfoDict, SpanType
+from . import models, emulator_message_processor
+from .. import messages
+from ...rest_api.types import span_write, trace_write
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LocalEmulatorMessageProcessor(
```

Comment:
Do we really need this subclass here?
Not sure I understand the purpose of introducing the third inheritance level. Maybe let's put everything in the base class and split only if we need it it in the future?